src/bindings/python/src/pyopenvino/graph/passes/pattern_ops.cpp

```diff
@@ -493,37 +493,36 @@ static void reg_pattern_optional(py::module m) {
                       }),
                       py::arg("type_name"),
                       R"(
-                      Create Optional with the given node type.
-
-                      :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
-                      :type type_names: List[str]
+        Create Optional with the given node type.
```

Comment:
```suggestion
        Create Optional with the given node type.

```

there is need empty line for proper documentation generation